.github/workflows/version_bump.yml

```diff
@@ -0,0 +1,91 @@
+name: Version bump
+
+on:
+  workflow_dispatch:
+    inputs:
+      bump:
+        description: "Version bump type"
+        required: true
+        type: choice
+        default: auto
+        options:
+        - auto
+        - patch
+        - minor
+        - major
+      prerelease:
+        description: "Increase to this prerelease version"
+        required: false
+        type: choice
+        default: none
+        options:
+        - none
+        - alpha
+        - beta
+        - rc
+
+jobs:
+  version_bump:
+
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+    permissions:
+      contents: write       # to create a github release
+      pull-requests: write  # to create and update PRs
+
+    steps:
+    - uses: actions/checkout@v3.0.2
+      with:
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v4.2.0
+      with:
+        python-version: "3.10"
+        architecture: x64
+
+    - name: Bump version
+      id: cz-bump
+      uses: commitizen-tools/commitizen-action@0.14.1
+      with:
+        increment: ${{ github.event.inputs.bump != 'auto' && github.event.inputs.bump || '' }}
+        prerelease: ${{ github.event.inputs.prerelease != 'none' && github.event.inputs.prerelease || '' }}
+        commit: "false"
+        push: "false"
+        changelog: "true"
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
This token has insufficient permissions.

Reply:
The `read:org` permission is required.

Reply:
@WillDaSilva yeah, do we need the Melty bot account in MeltanoLabs?

Reply:
We just need to add a secret to the org with a token with the proper permissions. I don't think it should be the MeltyBot token, since that one is quite powerful, and we have many third-party maintainers in this org. Any of them could extract the key we use for this with ease, and then use it to seriously damage the Meltano org.

If you'd like, I can make a new MeltyBot token with fewer permission, and then add it to this org.

Reply:
@WillDaSilva yeah, that'd be awesome 🙏